content/general/channel_lifecycle.textile

```diff
@@ -0,0 +1,124 @@
+
+h2(#configuration). Configuration
+
+Metachannels have defaults for various attributes - such as the maximum message rate - but certain attributes are also configurable on a per-application basis. These custom configurations are presently only available to "enterprise":https://www.ably.io/pricing/enterprise accounts, and we do not provide dashboard support for modifying that configuration; "contact us":https://www.ably.io/contact to discuss your requirements if this is relevant to you.
+
+The configuration options for the @channel.lifecycle@ metachannel specifically are:
+
+- enabled := _true_ determines whether or not the metachannel is enabled<br>__Type: @boolean@__
+- sendOccupancy := _false_ determines whether or not "occupancy":#occupancy events are enabled<br>__Type: @boolean@__
+- occupancyUpdateInterval := _5000_ where "occupancy":#occupancy events are enabled, the minimum interval, in milliseconds, between "occupancy":#occupancy updates on any given channel<br>__Type: @int@__
+
+h2(#realtime). Subscribing via a realtime connection
+
+The rate of messages published in a metachannel can exceed the rate that a realtime connection can handle. At present, realtime connections are permitted to subscribe to a metachannel but this may result in the connection being frequently detached as the outbound rate limit is hit on the connection. As the rate on a metachannel grows it eventually will scale by having its traffic sharded across multiple independent shard channels, each carrying a fraction of the traffic. Once a metachannel scales to a level that it is sharded, it is no longer possible to subscribe with a realtime connection; it is only possible to get messages from that metachannel via reactor rules.
+
+h2(#metadata-rest). Channel metadata REST API
+
+<%= partial 'shared/_channel_metadata' %>
+
+h2(#enumeration-rest). Channel enumeration REST API
+
+<%= partial 'shared/_channel_enumeration' %>
+
+h2(#reactor). Reactor rules
+
+You can associate "reactor rules":https://www.ably.io/reactor with metachannels in the same way as for regular channels; configure these using the "Message":/realtime/messages source and use a @channelFilter@ that matches the qualified metachannel name, or simply the name @[meta]channel.lifecycle@. The @Channel Lifecycle@ webhook source is no longer supported with the channel lifecycle metachannel.
+
+h1. Channel Lifecycle API Reference
+
+inline-toc.
+  Types:
+    - ChannelStatus#channel-status
+
+h2(#types). Types
+
+The primary payload of lifecycle events for channels is the "ChannelDetails":#channel-details type which contains the @channelId@ and other static information about the channel, plus a @status@ containing a "ChannelStatus":#channel-status instance which contains information about the current state of the channel.
```

Comment:
Is there another payload?  If not, isn't this just "The payload of lifecycle events..."